### ISSUE TITLE
Discord and Messenger do not show Notifications 

### ISSUE THREAD

**Issue by Alpheron** [getferdi/ferdi#1915]
Messenger shows notifications through Ferdi with a red dot over the icon but does not send notifications to the notification shade on macOS through a banner.

Discord does not show notifications through Ferdi and shows no red dot over the icon but also does not send notifications through the notification shade also on macOS.

**Comment by agreenbhm** [comment]
Same happening for me. v5.6.2

**Comment by mhatvan** [comment]
> @mhatvan great news! Would you mind linking the commit/pr? I can't seem to find it. Curious as to the fix.

https://github.com/getferdi/recipes/pull/734

Direct notifications probably stopped working due to changed DOM. Indirect notifications were never counted in the first place.

**Comment by agreenbhm** [comment]
Thanks so much!

**Comment by ylluminarious** [comment]
> Note that I can't test the macOS notification banner since I am not able to simulate an interaction with another person on Discord/Messenger easily right now.

@mhatvan @agreenbhm Is there anyone that could get this working as well? It's rather inconvenient to be missing these banners, and I haven't been able to get them working for a while now, including a number of beta and stable releases...

**Comment by Alpheron** [comment]
I am currently using Ferdi on a Mac, I should be able to give this a quick test, in a couple of hours. I will test it out and let you all know how this goes.

**Comment by vraravam** [comment]
download the nightly, replace the recipe, restart

**Comment by Alpheron** [comment]
@vraravam Thank you! I just got finished setting it up and should be able to let you know if any banners are showing. Expect an update soon!

**Comment by ylluminarious** [comment]
@Alpheron Yup, that's exactly the same behavior I've observed. Glad to know it's not just me...

**Comment by themagpieswarble** [comment]
Yeah, would love for this to work for channels even if it was only for ones that were currently in focus before switching Ferdi tabs to another messenger.
